packages/ffmpeg/src/video-frame-extractor.ts

```diff
@@ -5,6 +5,8 @@ import {getVideoCodec} from './utils';
 
 type VideoFrameExtractorState = 'processing' | 'done' | 'error';
 
+// TODO: investigate if jpeg option was correctly removed on main
```

Comment:
there will be a conflict with main right now. In main, png: true is hardcoded in ffmpeg-client.ts:

```
      import.meta.hot.send('revideo:ffmpeg-video-frame', {
        data: {
          id: id,
          filePath: src,
          startTime: time,
          duration,
          fps,
          png: true,
        }
```